themes/src/main/resources-community/theme/base/admin/messages/admin-messages_de.properties

```diff
@@ -0,0 +1,1513 @@
+#certificate.tooltip=Client Certificate for validate JWT issued by client and signed by Client private key from your keystore.
+#publicKey.tooltip=Public Key for validate JWT issued by client and signed by Client private key.
+#no-client-certificate-configured=No client certificate configured
+#gen-new-keys-and-cert=Generate new keys and certificate
+#import-certificate=Import Certificate
+#gen-client-private-key=Generate Client Private Key
+#generate-private-key=Generate Private Key
+#kid=Kid
+#kid.tooltip=KID (Key ID) of the client public key from imported JWKS.
+#use-jwks-url=Use JWKS URL
+#use-jwks-url.tooltip=If the switch is on, then client public keys will be downloaded from given JWKS URL. This allows great flexibility because new keys will be always re-downloaded again when client generates new keypair. If the switch is off, then public key (or certificate) from the Keycloak DB is used, so when client keypair changes, you always need to import new key (or certificate) to the Keycloak DB as well.
+#jwks-url=JWKS URL
+#jwks-url.tooltip=URL where client keys in JWK format are stored. See JWK specification for more details. If you use Keycloak client adapter with "jwt" credential, then you can use URL of your app with '/k_jwks' suffix. For example 'http://www.myhost.com/myapp/k_jwks' .
+#archive-format=Archive Format
+#archive-format.tooltip=Java keystore or PKCS12 archive format.
+#key-alias=Key Alias
+#key-alias.tooltip=Archive alias for your private key and certificate.
+#key-password=Key Password
+#key-password.tooltip=Password to access the private key in the archive
+#store-password=Store Password
+#store-password.tooltip=Password to access the archive itself
+#generate-and-download=Generate and Download
+#client-certificate-import=Client Certificate Import
+#import-client-certificate=Import Client Certificate
+#jwt-import.key-alias.tooltip=Archive alias for your certificate.
+#secret=Secret
+#regenerate-secret=Regenerate Secret
+#registrationAccessToken=Registration access token
+#registrationAccessToken.regenerate=Regenerate registration access token
+#registrationAccessToken.tooltip=The registration access token provides access for clients to the client registration service.
+add-role=Rolle hinzuf\u00FCgen
+role-name=Rollenname
+#composite=Composite
+description=Beschreibung
+no-client-roles-available=Keine Client-Rollen verf\u00FCgbar
+#composite-roles=Composite Roles
+#composite-roles.tooltip=When this role is (un)assigned to a user any role associated with it will be (un)assigned implicitly.
+realm-roles=Realm-Rollen
+available-roles=Verf\u00FCgbare Rollen
+add-selected=Gew\u00E4hlte hinzuf\u00FCgen
+#associated-roles=Associated Roles
+#composite.associated-realm-roles.tooltip=Realm level roles associated with this composite role.
+#composite.available-realm-roles.tooltip=Realm level roles that you can associate to this composite role.
+remove-selected=Gew\u00E4hltes entfernen
```

Comment:
Here you use "Gewähltes", in the message above you used "Gewählte". I think this should be consistent.

Reply:
Wouldn't you say "Ausgewählte" instead of "Gewählte"? But nevertheless I agree it should be consistent.